### PR TITLE
draft schedule

### PR DESCRIPTION
**Pull request by bast**
The page needs to be referred to from the index page but I will do that after the PR is in to avoid conflicts with the venue PR.

### CODE REVIEW

File: conference/schedule.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+layout: default
+---
+
+# Tentative conference schedule
+
+<div class="alert alert-dismissible alert-warning">
+  <p>
+    We are still working on refining the schedule so this is not final but we
+    thought it would be useful to share what we have so far.
+  </p>
```
Comment (rkdarst) [inline conference/schedule.md:11]:
```suggestion
  We will adapt to any interesting ideas that are proposed, so please
  discuss with us or submit your interesting ideas as events. 
  </p>
```

File: conference/schedule.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      <td>11:00 - 12:00</td>
+      <td>Lightning talks in plenary, before lunch, 5 min each</td>
+    </tr>
+    <tr>
+      <td>12:00 - 13:30</td>
+      <td>Lunch</td>
+    </tr>
+    <tr>
+      <td>13:30 - 15:00</td>
+      <td>Workshops / parallel Sessions (2 larger, 3 smaller)</td>
```
Comment (rkdarst) [inline conference/schedule.md:45]:
```suggestion
      <td>Workshops / parallel sessions (2 larger, 3 smaller)</td>
```

File: conference/schedule.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    </tr>
+    <tr>
+      <td>15:00 - 15:30</td>
+      <td>Afternoon break</td>
+    </tr>
+    <tr>
+      <td>15:30 - 17:00</td>
+      <td>Workshops / parallel Sessions (2 larger, 3 smaller)</td>
```
Comment (rkdarst) [inline conference/schedule.md:53]:
```suggestion
      <td>Workshops / parallel sessions (2 larger, 3 smaller)</td>
```

File: conference/schedule.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+      <td>10:45 - 12:00</td>
+      <td>Panel group session for all RSE groups present with Q&A</td>
+    </tr>
+    <tr>
+      <td>12:00 - 13:30</td>
+      <td>Lunch</td>
+    </tr>
+    <tr>
+      <td>13:30 - 15:00</td>
+      <td>Workshops / parallel Sessions (2 larger, 3 smaller)</td>
```
Comment (rkdarst) [inline conference/schedule.md:89]:
```suggestion
      <td>Workshops / parallel sessions (2 larger, 3 smaller)</td>
```

File: conference/schedule.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    </tr>
+    <tr>
+      <td>15:00 - 15:30</td>
+      <td>Afternoon break</td>
+    </tr>
+    <tr>
+      <td>15:30 - 16:30</td>
+      <td>Workshops / parallel Sessions (2 larger, 3 smaller)</td>
```
Comment (rkdarst) [inline conference/schedule.md:97]:
```suggestion
      <td>Workshops / parallel sessions (2 larger, 3 smaller)</td>
```

**Comment by bast** [comment]
I like the media/exhibits!

**Pull request merged**